lib/prettyoutput.js

```diff
@@ -55,7 +56,8 @@ internals.parseOptions = opts => {
         indentation: indentation,
         maxDepth: opts.maxDepth || 3,
         colors: !opts.noColor ? colors : null,
-        alignKeyValues: _.isBoolean(opts.alignKeyValues) ? opts.alignKeyValues : true
+        alignKeyValues: _.isBoolean(opts.alignKeyValues) ? opts.alignKeyValues : true,
+        hideUndefined: opts.hideUndefined ? true : false
```

Comment:
Sorry one last comment, can you please follow the structure alignKeyValues: _.isBoolean(opts.alignKeyValues) ? opts.alignKeyValues : true, ->
hideUndefined: _.isBoolean(opts.hideUndefined) ? opts.hideUndefined : true,